views/ministryteams.ejs

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html lang="en">
+  
+<%- include widgets/header.ejs %>
+<%- include widgets/mainNav.ejs %>
+<div class="overlay" id="our-beliefs">
+  <h1 id="top">Ministry Teams</h1>
+</div>
+
+
+<div class="container">
+  <div class="row">
+      <div class="col-md-8 mx-auto px-4 py-5 my-3 order-md-1">
+        <h3>List of Ministry Teams</h3>
+        <hr>
+        <h5 id = 0>
+        &nbsp&#9830
+        <% teams.forEach(function(Team, index){%>
+          <a href=#<%=index%>><%=Team.name%></a>&nbsp&#9830&nbsp
+        <%})%>
+        </h5>
+        <hr>
+        <br>
+        <h3>Music Team</h3>
+        <% teams.forEach(function(Team, index){%>
+          <hr>
+          <p><%=Team.description%></p>
+            <% if(user){%>
+              <p><%=Team.leader%></p>
+              <p><%=Team.contactInfo%></p>  
+            <%}else{%>
+              <p id=<%=index+1%>>Please login for contact information.<br><a href=#top>return to the top of the page</a></p>
```

Comment:
I don't think "return to the top of the page" is necessary for every single ministry team description. If you want, you could just put a single "return to top" at the bottom of the page after all of the descriptions.